tests/main/cifs-home/task.yaml

```diff
@@ -0,0 +1,118 @@
+summary: Test that snaps still work when /home is a CIFS mount
+
+details: |
+    Snapd now contains a feature where CIFS-mounted /home (or any sub-directory)
+    initializes a workaround mode where all snaps gain minimal amount of network
+    permissions sufficient for CIFS to operate.
+
+# Limit the test to systems that have samba fairly new samba.
+systems:
+  - ubuntu-22.04-*
+  - ubuntu-24.04-*
+
+prepare: |
+    # We don't expect to leak the test-remote user, group or mount.
+    tests.cleanup defer NOMATCH 'test-remote' /etc/passwd
+    tests.cleanup defer NOMATCH 'test-remote' /etc/group
+    tests.cleanup defer NOMATCH 'test-remote' /proc/self/mountinfo
+
+    # Install a package with additional kernel modules, so that we can mount cifs.
+    if ! tests.pkgs install "linux-modules-extra-$(uname -r)"; then
+      echo "SKIP: Kernel version and extras module mismatch"
+      exit 0
+    fi
+    tests.cleanup defer tests.pkgs remove "linux-modules-extra-$(uname -r)"
```

Comment:
btw. does tests.pkgs install check if the package is installed already? if it was, then perhaps we shouldn't be removing it unconditionally

Reply:
Done